tests/integration/veth_test.py

```diff
@@ -338,12 +359,19 @@ def bridges_with_port():
         ]
     }
     try:
-        libnmstate.apply(d_state)
+        apply_with_description(
+            "Configure the ovs bridge ovs-br0 to have the port veth1, "
+            "configure the linux bridge br0 to have the port veth1peer",
```

Comment:
veth1 with veth1peer as peer, assign veth1 to OVS bridge ovs-br0 and veth1peer to linux bridge br0.